### ISSUE TITLE
in PATCH v2/subcription if try to update throttling to 0 (default), it is not changed

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#2030]
in PATCH v2/subcription if try to update throttling to 0 (default), it is not changed in mongo.

#### update subcription request
```
curl -v -X PATCH http://localhost:1026/v2/subscriptions/pplication/json" -H "Fiware-Service: test_update_happy_path" -H "Fiware-ServicePath: /test"  -d '{ "notification": {  "callback": "http://localhost", "throttling": 0, "attributes": ["pepe"] }}'
```

#### update subcription response
```
HTTP 204 No Content
Connection: Keep-Alive
Content-Length: 0
Date: Tue, 12 Apr 2016 16:02:07 GMT
```

#### mongo doc
```
{ "_id" : ObjectId("570d0dcaeb4996bff55a0330"), "reference" : "http://localhost", "entities" : [ { "id" : "Room", "isPattern" : "false" } ], "attrs" : [ "pepe" ], "expiration" : NumberLong(1459864800), "servicePath" : "/test", "throttling" : NumberLong(5), "description" : "One subscription to rule them all", "status" : "active", "conditions" : [ { "type" : "ONCHANGE", "value" : [ "temperature " ] } ], "expression" : { "q" : "temperature>40", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "JSON" }

```

**Comment by fgalan** [comment]
Functest added in PR https://github.com/telefonicaid/fiware-orion/pull/2331 during the 1.3.0 development cycle. However, we have found that the bug was fixed in 1.2.0. Thus I've changed the milestone.

@iariasleon please have a look (in both 1.2.0 and develop) and close if you find it solved.

**Comment by iariasleon** [comment]
LGTM
Request updated
```
curl -v -X PATCH http://localhost:1026/v2/subscriptions/5790c0572b452ae6b944839d -H "Content-Type: application/json" -H "Fiware-Service: test_update_happy_path" -H "Fiware-ServicePath: /test"  -d '{ "throttling": 0}'
```
mongo doc before:
```
{ "_id" : ObjectId("5790c0572b452ae6b944839d"), "expiration" : NumberLong(1459864800), "reference" : "http://localhost", "custom" : false, "throttling" : NumberLong(5), "servicePath" : "/test", "description" : "One subscription to rule them all", "status" : "active", "entities" : [ { "id" : "Room", "isPattern" : "false" } ], "attrs" : [ ], "blacklist" : false, "conditions" : [ ], "expression" : { "q" : "temperature>40", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "normalized" }
```
mongo doc after:
```
{ "_id" : ObjectId("5790c0572b452ae6b944839d"), "expiration" : NumberLong(1459864800), "reference" : "http://localhost", "custom" : false, "throttling" : NumberLong(0), "servicePath" : "/test", "description" : "One subscription to rule them all", "status" : "active", "entities" : [ { "id" : "Room", "isPattern" : "false" } ], "attrs" : [ ], "blacklist" : false, "conditions" : [ ], "expression" : { "q" : "temperature>40", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "normalized" }

```
